packages/server/src/ui/layout/page-sidebar.css

```
@@ -69,3 +69,17 @@
   color: var(--inverted-text-color);
   text-decoration: none;
 }
+
+.page-sidebar__footer {
```

```suggestion
.page-sidebar__footer {
  position: absolute;
  left: 0;
  right: 0;
  bottom: var(--base-spacing);
  text-align: center;
}
```

maybe something like that?